### PR TITLE
 balancePID works with main

### PR DESCRIPTION
**Pull request by anisaa2326**
## Steps to Test
It was tested in sim, the balance functions are being called when x on the xboxcontroller is being pressed. Balance execute has a print statement, "trying to balance", this prints when x is presesd.

## Changelog
balancePID works with the news changes made to main.

### CODE REVIEW

**Review by Leaf-life — DISMISSED** [review]

**Review by DarthCooper — APPROVED** [review]

File: robots/configBasedRobot.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 import utils.configMapper
 
 class  ConfigBaseCommandRobot(commands2.TimedCommandRobot):
+    balanceing = False
```
Comment (raylmorris) [inline robots/configBasedRobot.py:12]: Is this still neede?

Comment (raylmorris) [reply robots/configBasedRobot.py:12]: It looks like it needs to be moved to `dumboBot.py`

Comment (anisaa2326) [reply robots/configBasedRobot.py]: I moved the balanceing = False to dumboBot and pushed

**Review by raylmorris — APPROVED** [review]
+1

**Review by DarthCooper — APPROVED** [review]

**Pull request merged**